js/client/modules/@arangodb/testing.js

```diff
@@ -86,6 +86,8 @@ let optionsDocumentation = [
   '',
   '   - `disableMonitor`: if set to true on windows, procdump will not be attached.',
   '   - `rr`: if set to true arangod instances are run with rr',
+  '   - `exceptionFilter`: on windows you can use this to abort tests on specific exceptions',
+  '                        i.e. `bad_cast` to abort on throwing of std::bad_cast',
```

Comment:
Please, add example here on how to set multiple exceptions: i.e. are they coma-separated and should be quoted?